### PR TITLE
fix: MD030/list-marker-space

### PR DESCRIPTION
**Pull request by nschonni**
All these except the last commit should show up as "whitespace only change".
I've included the regex's used to make the changes so you can redo it internally if you're intersted in the changes, but not the PR.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@nschonni : Thanks for your contribution! The author, @conceptdev, has been notified to review your proposed change.

**Comment by conceptdev** [comment]
Hey @nschonni, thanks for this! I'm not sure how I got tagged (versus all the other affected doc authors 😉) but I can tell you we generally have a file limit for PRs and this is probably too big to be merged... I'll take one of the repo admins to confirm.

Can you share the regex here in the comments? Even internally we couldn't run it over the entire docset at once (PR would still be too big to merge, per policy), but individual product owners or areas could gradually apply the fix.

**Comment by conceptdev** [comment]
@gewarren tagging you for visibility - this PR affects a number of docs/authors.

**Comment by nschonni** [comment]
@conceptdev the regexs are in each of the commit message bodies

File: docs/profiling/concurrency-visualizer.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ ms.workload:
 >
 > - Download the              [Concurrency Visualizer for Visual Studio 2017](https://marketplace.visualstudio.com/items?itemName=VisualStudioProductTeam.ConcurrencyVisualizer2017#overview) extension.
 > - Download the              [Concurrency Visualizer for Visual Studio 2015](https://marketplace.visualstudio.com/items?itemName=Diagnostics.ConcurrencyVisualizerforVisualStudio2015) extension.
->   -   Download the              [Concurrency Visualizer Collection Tools for Visual Studio 2015](http://www.microsoft.com/download/details.aspx?id=49103).
+>   - Download the              [Concurrency Visualizer Collection Tools for Visual Studio 2015](http://www.microsoft.com/download/details.aspx?id=49103).
```
Comment (gewarren) [inline docs/profiling/concurrency-visualizer.md]: Can you manually fix the indent here to be consistent?

**Comment by PRMerger10** [comment]
@nschonni : Thanks for your contribution! The author, @conceptdev, has been notified to review your proposed change.

File: docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md [hunk 2]
```diff
@@ -34,9 +34,9 @@ Refer to the [High DPI Desktop Application Development on Windows](https://docs.
 
 ## Enabling PMA
 To enable PMA in Visual Studio, the following requirements need to be met:
-1)  Windows 10 April 2018 Update (v1803, RS4) or later
-2)  .NET Framework 4.8 RTM or greater
-3)  Visual Studio 2019 with the ["Optimize rendering for screens with different pixel densities"](https://docs.microsoft.com/visualstudio/ide/reference/general-environment-options-dialog-box?view=vs-2019) option enabled
+1) Windows 10 April 2018 Update (v1803, RS4) or later
+2) .NET Framework 4.8 RTM or greater (currently ships as standalone preview or bundle with recent Windows Insider builds)
```
Comment (nschonni) [inline docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md]: Revert rebase conflict
```suggestion
2) .NET Framework 4.8 RTM or greater
```

**Comment by PRMerger12** [comment]
@nschonni : Thanks for your contribution! The author, @conceptdev, has been notified to review your proposed change.

File: docs/profiling/concurrency-visualizer.md (hunk 1 above)
Comment (nschonni) [reply docs/profiling/concurrency-visualizer.md]: Done

**Pull request merged**